users/migrations/0023_user_hubspot_sync_datetime.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.18 on 2023-05-23 10:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0022_backfill_userprofile_records"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="hubspot_sync_datetime",
+            field=models.DateTimeField(null=True),
```

Comment:
You might want to set a default value for this, or try an [approach like that described here](https://isotoma.com/blog/2015/11/23/sorting-querysets-with-nulls-in-django/) to get nulls returned first in the list.  When trying out locally, users with null values always come after users with non-null values.

Reply:
Good catch.  I found this https://docs.djangoproject.com/en/4.2/ref/models/expressions/#using-f-to-sort-null-values as well.